source/android/adaptivecards/src/androidTest/java/io/adaptivecards/objectmodel/ChoiceSetInputPropertiesTest.java

```diff
@@ -211,6 +229,72 @@ public ChoiceSetInput castTo(BaseCardElement element) {
         }
     }
 
+    @Test
+    public void isRequiredCompactValidation()
```

Comment:
Let's try to be a little more descriptive with our test case names + descriptions - is it possible to add some better description metadata for junit tests? If so let's start to add descriptions to explain at least what the test is actually trying to do. Also remember the most effective tests are those that test a single behavior in a given test as much as possible. It should be very obvious looking at the test name what hte test is testing and what the expected behavior is.